10up/block-components
# Replace `TextControl` with `SearchControl` in `ContentSearch` Component

### Description of the Change




### How to test the Change

1. Insert Content Picker or Post Picker block in /example 
2. Confirm search component is used and block is operating normally

### Changelog Entry


- Replaces TextControl with SearchControl in 'content-search' component
- Adds Content Picker block to test and demonstrate content-picker component
- Updates documentation to include new prop and updates demo gif


### Credits

Props @psorensen 


### Checklist:
